src/test/groovy/org/dormitory/autobotsoccub/EqualsVerifierShould.groovy

```diff
@@ -0,0 +1,17 @@
+package org.dormitory.autobotsoccub
+
+import nl.jqno.equalsverifier.EqualsVerifier
+import org.dormitory.autobotsoccub.engine.model.Game
+import org.dormitory.autobotsoccub.engine.model.GameData
+import spock.lang.Specification
+
+class EqualsVerifierShould extends Specification {
```

Comment:
Test class name should describe sut/test intention, e.g.:
EqualsAndHashcodeShould."be met by domain classes",
EqualsAndHashcodeTest."verify contract"

Reply:
I see, thank